client/widgets/project-health/health.directive.js

```diff
@@ -1,68 +1,16 @@
 angular.module('probedock.projectHealthWidget').directive('projectHealthWidget', function() {
   return {
     restrict: 'E',
-    controller: 'ProjectHealthWidgetCtrl',
     templateUrl: '/templates/widgets/project-health/health.template.html',
+    controller: 'ProjectHealthWidgetCtrl',
     scope: {
-      project: '='
+      organization: '=',
+      project: '=',
+      linkToVersion: '='
```

Comment:
Why is this called `linkToVersion` and not `linkable` like the others? What does it mean? It's not clear at all.